app/org/sagebionetworks/bridge/services/AuthenticationService.java

```diff
@@ -310,10 +311,18 @@ private UserSession getSessionFromAccount(Study study, CriteriaContext context,
         
         user.setSharingScope(optionsService.getEnum(healthCode, SHARING_SCOPE, SharingScope.class));
         user.setDataGroups(optionsService.getStringSet(healthCode, DATA_GROUPS));
+        user.setLanguages(optionsService.getOrderedStringSet(healthCode, LANGUAGES));
```

Comment:
JIRA or separate task for optionsService.getAllOptions() or something equivalent. This allows us to make one call to DDB that gets all options for a single user.